### PR TITLE
Changed process domain events

### PR DESCRIPTION
**Pull request by andrey-berenda**

### CODE REVIEW

File: tests/test_process_domain_events.py [hunk 1]
```diff
@@ -18,11 +20,16 @@ class CustomDomainEvent(DomainEvent):
...
         handled_domain_events = []
+        many_handled_domain_events = []
 
         @domain_event_handler
         def empty_handle(self, domain_event: CustomDomainEvent):
             self.handled_domain_events.append(domain_event)
 
+        @domain_event_handler
+        def handle_many(self, domain_events: typing.List[CustomDomainEvent]):
+            self.many_handled_domain_events.append(domain_events)
```
Comment (pristupa) [inline tests/test_process_domain_events.py]: handled_many_domain_events (or handled_multiple_domain_events)

File: tests/test_process_domain_events.py [hunk 2]
```diff
@@ -18,11 +20,16 @@ class CustomDomainEvent(DomainEvent):
 def empty_handler():
     class Handler:
         handled_domain_events = []
+        many_handled_domain_events = []
```
Comment (pristupa) [inline tests/test_process_domain_events.py]: handled_many_domain_events (or handled_multiple_domain_events)

File: tests/test_process_domain_events.py [hunk 3]
```diff
@@ -51,3 +58,4 @@ def test_process_domain_events(empty_handler):
 
     # Assert
     assert empty_handler.handled_domain_events == [domain_event]
+    assert empty_handler.many_handled_domain_events == [[domain_event]]
```
Comment (pristupa) [inline tests/test_process_domain_events.py]: handled_many_domain_events (or handled_multiple_domain_events)

**Review by pristupa — DISMISSED** [review]

**Review by pristupa — APPROVED** [review]

File: tests/test_process_domain_events.py [hunk 4]
```diff
@@ -51,3 +58,4 @@ def test_process_domain_events(empty_handler):
 
     # Assert
     assert empty_handler.handled_domain_events == [domain_event]
+    assert empty_handler.handled_many_domain_events == [[domain_event]]
```
Comment (mofr) [inline tests/test_process_domain_events.py:61]: Please check that multiple events are group into a single list.

Comment (andrey-berenda) [reply tests/test_process_domain_events.py:61]: today I will create PR and add this tests

Comment (andrey-berenda) [reply tests/test_process_domain_events.py:61]: https://github.com/pristupa/ddd/pull/9

**Pull request merged**